src/components/Board.js

```diff
@@ -28,22 +29,25 @@ const Board = (props) => {
   }, [props.boardName]);
 
   const cardComponents = () => {
-    return cards.map((card) => {
+    return cards.map((cardh) => {
```

Comment:
cardh refers to the hash card. 
the data structure is cards [ ], but inside the cards [ ] is a hash called card{ } which has the id, text, emoji.